submissions/nallikaea/html-css-popup/css/main.css

```diff
@@ -0,0 +1,254 @@
+* {
+  font-family:'Nanum Gothic', sans-serif;
+  text-decoration:none;
+  padding: 0 0;
+  margin: 0 0;
+}
+
+body {
+background-image: url(../popup-icons/background.png);
+background-size: cover;
+}
+
+#navigation {
+  display:inline-flex;
+  width:100%;
+  height:35px;
+  flex-direction:row-reverse;
+  align-items:center;
+  background-color:hsla(0, 0%, 0%, 0.8);
+}
+
+.text-nav-item{
+  color: hsl(0, 0%, 70%);
+  font-family: 'Nanum Gothic', sans-serif;
+  font-size: 16px;
+}
+
+.menu-button, .more_button{
+  display: none;
```

Comment:
This elements did not have a focus and did not open by space.
Try `display: none;` instead use:
```
opacity: 0;
position: absolute;